src/go/k8s/tests/e2e/additional-configuration/03-verify-config-changed.yaml

```diff
@@ -1,4 +1,5 @@
 apiVersion: kuttl.dev/v1beta1
 kind: TestStep
 commands:
-- command: ./verify-config.sh 11 5
+- command: ./verify-config-v22.1.sh 11 5
```

Comment:
The second argument 2 is unnecessary. The crash loop limit is not used in the 00 step